marklogic-client-api/src/test/java/com/marklogic/client/test/StringSearchTest.java

```diff
@@ -107,6 +112,30 @@ public void testStringSearch2() throws IOException {
     }
   }
 
+	@Test
+	public void testStringSearch3() throws IOException {
+		TextDocumentManager textDocumentManager = Common.client.newTextDocumentManager();
+		DocumentWriteSet batch = textDocumentManager.newWriteSet();
+		int numberOfDocs = 100;
+		String[] uris = new String[numberOfDocs];
+		for(int i=0; i<numberOfDocs;i++){
+			uris[i]="/test/forTesting/"+i+".txt";
+			batch.add(uris[i], new StringHandle("Text for doc -"+i).withFormat(Format.TEXT));
+		}
+		textDocumentManager.delete(uris);
+		textDocumentManager.write(batch);
+		QueryManager queryMgr = Common.client.newQueryManager();
+		queryMgr.setPageLength(200);
```

Comment:
I just tried this out - call `setPageLength` on the `textDocumentManager`, not on the `queryManager`.